api/app/Policies/ClassificationPolicy.php

```diff
@@ -20,7 +19,7 @@ class ClassificationPolicy
      */
     public function viewAny(?User $user)
     {
-        return true;
+        return $user->isAbleTo("view-any-classification");
```

Comment:
For each of these, user can be null so we need some null safe operators for these:

```php
  return $user?->isAbleTo("view-any-classification");
```